src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -497,6 +497,14 @@ def get_raw_token(self, resource=None, scopes=None, subscription=None, tenant=No
                 None if tenant else str(account[_SUBSCRIPTION_ID]),
                 str(tenant if tenant else account[_TENANT_ID]))
 
+    def get_msal_token(self, scopes, data):
+        """Get VM SSH certificate. Do not use it for other purposes. To get an access token, use get_raw_token instead.
+        """
+        credential, _, _ = self.get_login_credentials()
+        certificate_string = credential.get_token(*scopes, data=data).token
```

Comment:
`get_token` will be replaces by `acquire_token` after migrating managed identity authentication to MSAL: https://github.com/Azure/azure-cli/pull/25959